tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,48 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+const WSTRING general_error_message = WStr("Failed to instrument the method.");
+
+inline WSTRING GetGenericErrorMessageWithErrorCode(short errorCode)
+{
+    return general_error_message + WStr(" [Error Code: ") + shared::ToWSTRING(errorCode) + WStr("]");
+}
+
+const WSTRING invalid_probe_method_already_instrumented =
+    WStr("Dynamic Instrumentation failed to install the probe because the corresponding method is already instrumented by another product.");
+const WSTRING invalid_method_probe_probe_is_not_supported =
+    WStr("The method where the probe should have been placed is not supported for now.");
+const WSTRING line_probe_il_offset_lookup_failure =
+    WStr("There was a failure in determining the exact location where the line probe was supposed to be placed.");
+const WSTRING line_probe_in_async_generic_method_in_optimized_code =
+    WStr("Placing line probes in async generic methods in Release builds is currently not supported. Please note that this limitation only applies to optimized code.");
+const WSTRING invalid_probe_failed_to_instrument_method_probe = 
+    GetGenericErrorMessageWithErrorCode(1);
+const WSTRING invalid_probe_failed_to_instrument_line_probe =
+    GetGenericErrorMessageWithErrorCode(2);
+const WSTRING invalid_probe_probe_cctor_ctor_not_supported =
+    WStr("Instrumentation of constructors are not supported for now.");
```

Comment:
Can't we instrument ctor? (not static one)

Reply:
Technically we can but a while ago we decided to be more conservative about this.
Once the Fault-Tolerant Instrumentation will be implemented, we will re-evaluate if we want to allow c'tor instrumentation as well.